hw/bsp/bbc_microbit/src/sbrk.c

```diff
@@ -17,6 +17,8 @@
  * under the License.
  */
 
+#include <hal/hal_bsp.h>
+
```

Comment:
This include is unnecessary. Given that, it's better not to make this change.

If the other BSP has this one in it, it would be better to remove it from there as well.